Lesson_03/solution_task_n1.py

```diff
@@ -0,0 +1,33 @@
+# 1. Create a program that will expect the user to enter text and output information for each typed character:
+#
+# - this "number" + what it is (even, odd),
+# - this is a "string" + what is it (uppercase or lowercase),
+# - is a "symbol"
+
+
+def check_chars(input_string):
+
+    for char in input_string:
+
+        if char.isdigit() == True:
+            print(f"{char} is a digit", end=" ")
+
+            if int(char) % 2 == 0:
+                print(f"and it's an even number")
+            else:
+                print(f"and it's an odd number")\
+
+        elif char.isalpha() == True:
```

Comment:
You can skip `== True` since if will make this check in the end. So it can be simplified to `elif char.isalpha():`